packages/integration-tests/src/entities/AuthorCodegen.ts

```diff
@@ -109,7 +109,7 @@ export interface AuthorOpts {
   graduated?: Date | null;
   wasEverPopular?: boolean | null;
   address?: Address | null;
-  businessAddress?: z.infer<typeof AddressSchema> | null;
+  businessAddress?: z.input<typeof AddressSchema> | null;
```

Comment:
Did these sneak in from a unrelated/prior change? Or maybe they were missing from a prior change, so bringing this file up to date? Just checking.

Reply:
Ah, yup! I this is bringing up the codegen to date - suspect I didn't re-run codegen after changing the zod fieldType.

Happy to move to a separate PR if you would like to keep this one clean.

Reply:
Np, sneaking it in here is just fine. We should eventually get CI setup to detect diffs in the codegen output--we've got that on our internal repos but haven't set it up here yet.